USAGE.md

```diff
@@ -290,3 +247,11 @@ require:
 `packwerk` will directly call `require` with these paths.
 You can prefix local files with a dot to define them relative to `packwerk.yml`, or you can use absolute paths.
 You can also reference the name of a gem.
+
+## Examples
+### Privacy Checker
+[`packwerk-extensions`](https://github.com/rubyatscale/packwerk-extensions) (originally extracted from `packwerk`) can be used to help define and enforce public API boundaries of a package. See the README.md for more details. To use this, add it to your `Gemfile` and then require it via `packwerk.yml`:
```

Comment:
This is something I discussed with @mclark . Originally we called the gem `packwerk-privacy` but he made a good point that (A) clients would likely not want to install a bunch of different gems to be able to use extra checkers and (B) from a maintenance perspective it's simpler to maintain a repo of several checkers than have one repo for each checker. The analogy he used was rubocop, or different rubocop enhancement gems which ship with several cops.

Reply:
Understood.

> clients would likely not want to install a bunch of different gems to be able to use extra checkers

I know what you mean, and borrowing conventions from rubocop, extension gems often ship with a collection of cops instead of just one. I think what I'm concerned about more is the name `packwerk-extensions` because it sounds too all-encompassing. I would assume there will be extensions built in the future that aren't in this package.

>  from a maintenance perspective it's simpler to maintain a repo of several checkers than have one repo for each checker.

Yes, but you can do that in a monorepo of separately released packages. If you only have one checker in mind there's nothing stopping you from making a gem that includes several other more specific gems later on (this is basically what Rails is). If you find a bug in one of the checkers, you can fix, bump, and release the specific checker and not be blocked by work being done in other checkers.

What I'm getting at is, I wouldn't expect `packwerk-extensions` to ship with just one checker, and I think the theming around the gem name should be thought of before you release it. I ultimately have no say in this though, so feel free to do what makes sense to you!